live-ingester/src/integration-test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/e2e/repository/RequestFiltersIntegrationTest.java

```diff
@@ -0,0 +1,481 @@
+                    "resource": {
+                      "@type": "NodeResource",
+                      "id": "d71dd823-82c7-477c-8490-04cb0e826e04",
+                      "name": "purchase-order-scan.pdf",
+                      "nodeType": "cm:content",
+                      "createdByUser": {
+                        "id": "admin",
+                        "displayName": "Administrator"
+                      },
+                      "createdAt": "2024-03-02T11:14:15.695Z",
+                      "modifiedByUser": {
+                        "id": "abeecher",
+                        "displayName": "Alice Beecher"
+                      },
+                      "modifiedAt": "2024-03-06T10:29:42.529Z",
+                      "content": {
+                        "mimeType": "application/pdf",
+                        "sizeInBytes": 531152,
+                        "encoding": "UTF-8"
+                      },
+                      "properties": {
+                        "cm:title": "Purchase Order",
+                        "cm:versionType": null,
+                        "cm:versionLabel": "1.0",
+                        "cm:taggable": null
+                      },
+                      "aspectNames": [ "cm:versionable", "cm:author", "cm:titled" ],
+                      "isFolder": false,
+                      "isFile": true
+                    },
+                    "resourceBefore": {
+                      "@type": "NodeResource",
+                      "modifiedAt": "2024-03-02T11:14:25.223Z",
+                      "modifiedByUser": {
+                        "id": "admin",
+                        "displayName": "Administrator"
+                      },
+                      "properties": {
+                        "cm:title": null,
+                        "cm:versionType": "MAJOR",
+                        "cm:versionLabel": "1.0",
+                        "cm:taggable": null,
+                        "cm:description": "Old Description"
+                      },
+                      "aspectNames": [ "cm:versionable", "cm:thumbnailModification", "cm:author" ]
+                    }
+                  }
+                }""";
+        // when
+        containerSupport.raiseRepoEvent(repoEvent);
+
+        // then
+        String expectedBody = """
+                {
+                  "objectId" : "d71dd823-82c7-477c-8490-04cb0e826e04",
+                  "eventType" : "update",
+                  "properties" : {
+                    "cm:title" : "Purchase Order",
+                    "aspectsNames" : [ "cm:versionable", "cm:author", "cm:titled" ],
+                    "modifiedBy" : "abeecher"
+                  },
+                  "removedProperties" : [ "cm:versionType", "cm:description" ]
+                }""";
+        containerSupport.expectHxIngestMessageReceived(expectedBody);
+    }
+
+    @Test
+    @Order(5)
+    void testUpdateRequestWithAspectInDeniedFilter()
```

Comment:
I guess for completeness we could add a test where `sc:secured` is removed from the node. We'd need to talk with the HxI team to determine whether they need us to send a `create` event or if we can get away with an `update` event for this.

Reply:
I thought about that while implementing the feature but assumed that this will come to resolution sometime later when we exactly know how we want to deal with such events as all other filters will have similar results.